src/main/java/uk/gov/hmcts/sptribs/common/AddSystemUpdateRole.java

```diff
@@ -15,17 +15,16 @@ public class AddSystemUpdateRole {
 
     public List<UserRole> addIfConfiguredForEnvironment(List<UserRole> userRoles) {
         List<UserRole> existingRoles = new ArrayList<>(userRoles);
-        String environment = System.getenv().getOrDefault("ENVIRONMENT", null);
 
-        if (null != environment && environment.equalsIgnoreCase(ENVIRONMENT_AAT)) {
+        if (isEnvironmentAat()) {
             existingRoles.add(SYSTEMUPDATE);
         }
 
         return existingRoles;
     }
 
-    public boolean isEnvironmentAat() {
-        String environment = System.getenv().getOrDefault("ENVIRONMENT", null);
-        return null != environment && environment.equalsIgnoreCase(ENVIRONMENT_AAT);
+    private boolean isEnvironmentAat() {
+        final String currentEnv = System.getenv().getOrDefault("ENVIRONMENT", null);
+        return null != currentEnv && currentEnv.equalsIgnoreCase(ENVIRONMENT_AAT);
```

Comment:
I think it would be more readable if it was currentEnv != null instead of null != currentEnv

Reply:
Yes, I too agree.

This is the way most/all of the null checks in this project were implemented. So, I would like to speak to Tom on which way he wants us to proceed. If he too agrees with us, we can change all such occurrences in our project in one sitting. So, we can leave this as it is for now.